src/preFilter.ts

```diff
@@ -70,11 +70,14 @@ export async function filterTimestamps(searchee: Searchee): Promise<boolean> {
 			"timestamp.indexer_id": "indexer.id",
 			"timestamp.searchee_id": "searchee.id",
 		})
+		.where("searchee.name",
+		searchee.name
+		)
 		.whereIn(
 			"indexer.id",
 			enabledIndexers.map((i) => i.id)
 		)
-		.max({
+		.min({
```

Comment:
I don't think this change is right. it would probably make more sense called `first_searched_any` though…

Reply:
nvm I think I'm just getting confused

Reply:
ok yeah I think I understand it now. we really want first_searched_any and last_searched_all, and they are both calculated with `min`

Reply:
Ah, yes, that's a good name change. Let me just push that over.